INSTALL.md

```diff
@@ -108,23 +108,50 @@ The following contains scripts which will run all the required processes in orde
-Matrix      +------------------+   |        :7773               query API         | dendrite-room-server |--DB:roomserver
-Clients --->| client-api-proxy |---+                           +----------------->+----------------------+
-            +------------------+   |                           |               :7770           ^^
-          :8008                    | CS API   +----------------------------+                   ||
-                                   +--------->| dendrite-client-api-server |===================||
-                                   |          +----------------------------+   roomserver_input_topic_dev
-                                   |        :7771
-                                   |
-                                   | /media   +---------------------------+ 
+                                     /media   +---------------------------+ 
                                    +--------->| dendrite-media-api-server |
-                                              +---------------------------+
-                                            :7774
+                                   |          +---------------------------+
+                                   |        :7774
+                                   |
+                                   |
+                                   |/directory+----------------------------------+
+                                   +--------->| dendrite-public-rooms-api-server |<========++
+                                   |          +----------------------------------+         ||
+                                   |        :7775    |                                     ||
+                                   |    +<-----------+                                     ||
+                                   |    |                                                  ||
+                                   | /sync    +--------------------------+                 ||
+                                   +--------->| dendrite-sync-api-server |<================++
+                                   |    |     +--------------------------+                 ||
+Matrix      +------------------+   |    |   :7773    |                                     ||
+Clients --->| client-api-proxy |---+    +<-----------+                                     ||
+            +------------------+   |    |                                                  ||
+          :8008                    | CS API   +----------------------------+               ||
+                                   +--------->| dendrite-client-api-server |==++           ||
+                                        |     +----------------------------+  ||           ||
+                                        |   :7771    |                        ||           ||
+                                        |            |            client_data ||           ||
+                                        +<-----------+         ++=============++           ||
```

Comment:
I think the client_data stream goes to the sync-api-server not the room-server

Reply:
So it does. fixed in 46e0a66